### ISSUE TITLE
Tree items' text in events->configuration has insufficient contrast against its background in 'focused' state

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#867]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![14 1](https://user-images.githubusercontent.com/22068434/88355901-fca90300-cd1a-11ea-9143-eb9ab708026f.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
This shouldn't be closed yet. Reopening

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release found at https://github.com/Microsoft/accessibility-insights-windows/releases. Please validate and close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1388.01. All 6 modes are now compliant
